coordinator/src/test/scala/filodb.coordinator/queryplanner/LongTimeRangePlannerSpec.scala

```diff
@@ -263,14 +263,36 @@ class LongTimeRangePlannerSpec extends AnyFunSpec with Matchers with PlanValidat
 
   }
 
-  it("should direct raw-data queries to both raw planner only irrespective of time length") {
+  it("should direct instant raw-data queries with lookback to raw cluster when only need raw data"){
+    val start = now/1000
+    val step = 1.second.toSeconds
 
-    Seq(5, 10, 20).foreach { t =>
-      val logicalPlan = Parser.queryToLogicalPlan(s"foo[${t}m]", now, 1000)
-      val ep = longTermPlanner.materialize(logicalPlan, QueryContext()).asInstanceOf[MockExecPlan]
-      ep.name shouldEqual "raw"
-      ep.lp shouldEqual logicalPlan
-    }
+    // raw retention is 10m
+    val logicalPlan = Parser.queryToLogicalPlan("foo[9m]", start, step)
+    val ep = longTermPlanner.materialize(logicalPlan, QueryContext()).asInstanceOf[MockExecPlan]
+    ep.name shouldEqual "raw"
+    ep.lp shouldEqual logicalPlan
+
+    val logicalPlan2 = Parser.queryToLogicalPlan("foo[4m] offset 5m", start, step)
+    val ep2 = longTermPlanner.materialize(logicalPlan2, QueryContext()).asInstanceOf[MockExecPlan]
+    ep2.name shouldEqual "raw"
+    ep2.lp shouldEqual logicalPlan2
+  }
+
+  it("should direct instant raw-data queries with lookback to downSample cluster when range is over raw retention"){
```

Comment:
Toss a `foo[10m] offset 20m` in here, too -- just something that completely routes to the downsampled cluster.